bigquery/bqml/requirements.txt

```
@@ -3,6 +3,6 @@ google-cloud-bigquery-storage==2.27.0
 pandas==1.3.5; python_version == '3.7'
 pandas==2.0.3; python_version == '3.8'
 pandas==2.2.2; python_version > '3.8'
-pyarrow==17.0.0
+pyarrow==19.0.0
```

pyarrow dropped support for earlier versions 
```suggestion
pyarrow==17.0.0; python_version <= '3.8'
pyarrow==19.0.0; python_version > '3.9'
```